internal/write/write_configs.go

```diff
@@ -67,6 +67,9 @@ func WriteConfigs(halPath string, dir string) error {
 	if err := write(services.GetRosco(), filepath.Join(dir, "rosco.yml")); err != nil {
 		return err
 	}
+	if err := write(services.GetDeck(), filepath.Join(dir, "deck.yml")); err != nil {
```

Comment:
Wait, do we need this?

Reply:
Whoops...we do not!  I added this to help with debugging a test and forgot to remove it.  Thanks for catching it!